### PR TITLE
[Snyk] Upgrade: @material-ui/core, @material-ui/icons

### PR DESCRIPTION
**Pull request by marcoecv85**
<p>This PR was automatically created by Snyk using the credentials of a real user.</p><br /><h3>Snyk has created this PR to upgrade multiple dependencies.</h3>
👯‍♂ The following dependencies are linked and will therefore be updated together.
</br></br>
:information_source: Keep your dependencies up-to-date. This makes it easier to fix existing vulnerabilities and to more quickly identify and fix newly disclosed vulnerabilities when they affect your project.
</br></br>

 Name         | Versions     | Released on
:-------------|:-------------|:-------------
**@material-ui/core**</br>from 4.11.0 to 4.12.4 | **11 versions** ahead of your current version | **a year ago**</br>on 2022-04-03
**@material-ui/icons**</br>from 4.9.1 to 4.11.3 | **2 versions** ahead of your current version | **a year ago**</br>on 2022-04-03

<hr/>

**Note:** *You are seeing this because you or someone else with access to this repository has authorized Snyk to open upgrade PRs.*

For more information:  

🧐 [View latest project report](https://app.snyk.io/org/marco.calderon/project/b2848c4f-5685-4f01-a776-5c8e8f95ff72?utm_source&#x3D;github&amp;utm_medium&#x3D;referral&amp;page&#x3D;upgrade-pr)

🛠 [Adjust upgrade PR settings](https://app.snyk.io/org/marco.calderon/project/b2848c4f-5685-4f01-a776-5c8e8f95ff72/settings/integration?utm_source&#x3D;github&amp;utm_medium&#x3D;referral&amp;page&#x3D;upgrade-pr)

🔕 [Ignore this dependency or unsubscribe from future upgrade PRs](https://app.snyk.io/org/marco.calderon/project/b2848c4f-5685-4f01-a776-5c8e8f95ff72/settings/integration?pkg&#x3D;@material-ui/core&amp;pkg&#x3D;@material-ui/icons&amp;utm_source&#x3D;github&amp;utm_medium&#x3D;referral&amp;page&#x3D;upgrade-pr#auto-dep-upgrades)

### CODE REVIEW

**Review by seanadkinson — APPROVED** [review]
After you merge all these PRs, let's run the example Embedded SSO project (if you haven't already) and make sure things still work.  Follow the README to run the project (update it with any details that our customers might find useful), and I can assist if you get blocked.

Thanks!

**Review by giantryansaul — APPROVED** [review]

**Comment by giantryansaul** [comment]
I approved, but if this is an automated PR and hasn't been tested it should probably be closed instead.

**Pull request closed without merging**
